spec/integration/atomic_switcher_spec.rb

```diff
@@ -7,6 +7,7 @@
 require 'lhm/migration'
 require 'lhm/atomic_switcher'
 
+
```

Comment:
moar more better!  dunno git in while doing some git things since the forks are so out of whack, will remove.